python/ray/dashboard/metrics_exporter/client.py

```diff
@@ -25,6 +26,7 @@ class MetricsExportClient:
 
     def __init__(self, address, dashboard_controller, dashboard_id, exporter):
         self.dashboard_id = dashboard_id
+        self.address = address
```

Comment:
Assert check the web protocol?

Reply:
Yeah that's a good idea.

Reply:
I think it is not that important for now. I will add a check once it is exposed to users.